src/utils/subscriptions.js

```diff
@@ -3,7 +3,7 @@ const config = require('config');
 const { v4: uuidv4 } = require('uuid');
 const db = require('../storage/DataAccess');
 
-const fhirServerConfig = config.get('fhirServerConfig');
+const fhirServerConfig = config.fhirServerConfig;
```

Comment:
https://github.com/mcode/subscription-proxy/pull/7#discussion_r612419812

I think I initially had it like this and @blangley28 was getting an error.

Reply:
This is how it originally was on master 🤷‍♂️

Reply:
Well I think on `master` it was `const fhirServerConfig = config.get('fhirServerConfig').resolve();` in both `index.js` and `subscriptions.js` but `sendNotification` wasn't being used anywhere yet so we didn't get the error of the config being resolved in 2 places.

Reply:
Does the `config.get('fhirServerConfig')` not work?

Reply:
When I do that the tests break, in a way that doesn't really make sense:

```
FAIL  test/auth/controller.test.js (11.918s)
  loading express
    ✕ responds to token url (6344ms)
    ✕ responds to bad token url call (5014ms)

  ● loading express › responds to token url

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      39 |   });
      40 |
    > 41 |   it('responds to token url', function testPath(done) {
         |   ^
      42 |     request(server)
      43 |       .post('/auth/token')
      44 |       .send({

      at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
      at Suite.it (test/auth/controller.test.js:41:3)

  ● loading express › responds to bad token url call

    Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.

      51 |   });
      52 |
    > 53 |   it('responds to bad token url call', function testPath(done) {
         |   ^
      54 |     request(server)
      55 |       .post('/auth/token')
      56 |       .send({

      at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
      at Suite.it (test/auth/controller.test.js:53:3)

Test Suites: 1 failed, 1 total
Tests:       2 failed, 2 total
Snapshots:   0 total
Time:        11.957s
```

Reply:
Do notifications get sent correctly with `config.fhirServerConfig`?

Reply:
I just tested and with the current change to be `config.fhirServerConfig` notifications are not sent correctly (but everything else _seems_ to be working fine)

Reply:
I reverted this back to how it was prior to this PR, so hopefully nothing else broke